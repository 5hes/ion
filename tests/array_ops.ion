let array = [ 4 4 5 5 5 6 6 6 6 ]
echo @array
let array ++= [ 1 2 3 ]
echo @array
let array ::= [ 1 2 3 ]
echo @array
let array \\= [ 4 5 6 ]
echo @array

let array = [ 1 2 2 3 3 3 ]
echo @array
let array ++= 4
echo @array
let array ::= 0
echo @array
let array \\= 3
echo @array
