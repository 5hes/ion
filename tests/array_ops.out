4 4 5 5 5 6 6 6 6
4 4 5 5 5 6 6 6 6 1 2 3
1 2 3 4 4 5 5 5 6 6 6 6 1 2 3
1 2 3 1 2 3
1 2 2 3 3 3
1 2 2 3 3 3 4
0 1 2 2 3 3 3 4
0 1 2 2 4
